lib/article/index.js

```diff
@@ -5,7 +5,7 @@ require('./style.css');
 export default class Article extends Component {
   render () {
     return (
-        <div className='articleContainer'>
+        <div className='articleContainer' onClick={() => {window.scrollTo(0, 0);this.props.viewArticle(this.props.content)}}>
```

Comment:
what does the window.scrollTo(0,0) do? Also for readability, you can put each prop on a separate line.